python_ta/reporters/templates/template.txt

```diff
@@ -1,12 +1,105 @@
 <html>
 <head>
+    <link rel="stylesheet" type="text/css" href="stylesheet.css">
 </head>
 <body>
-<u1>
-    {% for message in messages %}
-        <p>{{ message.msg_id }} ({{ message.symbol }}) {{ message.obj }}<br/>&emsp;&emsp;{{message.msg }}</p>
-    {% endfor %}
+	
+		<!-- Constantly updated date:
+		<script type="text/javascript">
```

Comment:
This date isn't necessary; it's only important to see the time the report was generated, not the current time.